maint/ManyConfigTests

```diff
@@ -115,7 +115,7 @@ ISGCC=0
 
 # If the compiler is GCC or Clang, add a lot of warning switches.
 
-CC_VER_OUTPUT=`printf '#ifdef __GNUC__\nGCC=yes\n#endif\n' | $CC -E -`
+CC_VER_OUTPUT=`printf '#if defined(__GNUC__) && !defined(__clang__)\nGCC=yes\n#endif\n' | $CC -E -`
```

Comment:
It doesn't work? That's annoying, sorry.

We should fix the comment above to remove "or Clang".

Reply:
It might be broken also with Intel's (at least the legacy) compiler as well then.